main.js

```diff
@@ -0,0 +1,11 @@
+const navBar = document.querySelector('#desktop-nav');
+const hamburguerButton = document.querySelector('.menu-bars');
+const hamburguerIcon = document.querySelector('.navbar-icon');
+
+function doSomething() {
+  hamburguerIcon.classList.toggle('fa-bars');
+  hamburguerIcon.classList.toggle('fa-times');
+  navBar.classList.toggle('show-me');
```

Comment:
- [ ] When clicking a link in the mobile menu it does take me to the correct part of the page. But the menu keeps "open" at the top of the page. I think it would be better for the user experience if the mobile menu disappears when clicking on a link.

![image](https://user-images.githubusercontent.com/81522435/135772660-bb2ad284-4655-4155-b162-f54e0fba5c7b.png)